crates/aac/src/lib.rs

```diff
@@ -8,6 +8,9 @@
 //!
 //! `SPDX-License-Identifier: MIT OR Apache-2.0`
 #![cfg_attr(all(coverage_nightly, test), feature(coverage_attribute))]
+#![deny(missing_docs)]
+#![deny(unsafe_code)]
+#![deny(clippy::missing_const_for_fn)]
```

Comment:
let me change the ticket to get rid of this one. This one is a double edged sword. Lets not use const fn unless something we know is const. I mentioned this in the other pr #370 

> ive been thinking abt this lint. its a double edged sword, on one point its nice to have functions that are const fn since therefore they can be used in const contexts. On the other hand it means we are locking ourselves into this if we change the fn implementation and makes it so we cant use a const fn it is a breaking change. I think we should remove this and avoid using const fn unless we can 100% justify that the function can never be non-const.